src/components/FileReader.vue

```diff
@@ -0,0 +1,68 @@
+<script lang="ts" setup>
+import { ref, reactive, computed } from 'vue'
+import { useImportStore } from '@/stores/import'
+import type MatlabFile from "@/interfaces/IExpectedFileContent"
+
+const docFile: any = ref(null)
+const importFile: any = useImportStore()
+
+const file: any = reactive({})
+const content: any = ref("")
+const loading: any = ref(false)
```

Comment:
Please don't use any. Give these proper types. Does Vue not have a types package? Usually `@types/{name of package}`

Reply:
https://vuejs.org/guide/typescript/composition-api.html here

But this is not the issue, the file reader library from `JS` has some really funky types and it is hard to truly define it. I looked a bit into it but don't think it is worth the effort, will open ticket.

Reply:
see: https://github.com/Baby-Monitor-Simulator/s6-baby-monitor-simulator-webui/issues/13